app/views/layouts/_navbar.html.erb

```diff
@@ -1,12 +1,37 @@
-<% if user_signed_in? %>
-<%= image_tag current_user.image_link %>
-Welcome: <%= current_user.name.capitalize%>
-<%= link_to 'Home', root_path%>
-<%= link_to 'All users', users_path%>
-<%= link_to 'Create new post', new_post_path%>
-<%= link_to 'Edit account', edit_user_registration_path%>
-<%= link_to 'Profile', user_path(current_user)%>
-<%= link_to 'Friends', users_path(filter: 'friends')%>
-<%= link_to 'Notifications', users_path(filter: 'pending')%>
-<%= link_to 'Logout', destroy_user_session_path, method: :delete%>
-<% end %>
\ No newline at end of file
+<nav class="navbar navbar-expand-lg navbar-light bg-teal py-1">
+  <a class="navbar-brand" href="/">Final project</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav"
+    aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse d-flex justify-content-end" id="navbarNav">
+    <ul class="navbar-nav">
+      <li class="nav-item">
+        <%= link_to 'Home', root_path, class:"nav-link"%>
+      </li>
+      <li class="nav-item">
+        <%= link_to 'All users',users_path, class:"nav-link"%>
+      </li>
+      <li class="nav-item">
+        <%= link_to 'Edit account', edit_user_registration_path, class:"nav-link"%>
+      </li>
+      <li class="nav-item">
+        <%= link_to 'Profile', user_path(current_user), class:"nav-link"%>
+      </li>
+      <li class="nav-item">
+        <%= link_to 'Friends', users_path(filter: 'friends'), class:"nav-link"%>
+      </li>
+      <li class="nav-item">
+        <%= link_to 'Notifications', users_path(filter: 'pending'), class:"nav-link"%>
```

Comment:
[OPTIONAL] it would be nice if the notification showed an indication when there is a friend request pending